templates/jarvice-kns-scheduler.yaml

```diff
@@ -102,7 +102,7 @@ spec:
       - name: jarvice-kns-scheduler
         command: ["python3"]
         args: ["/main", "Nested"]
-        image: {{ include "jarvice.registry" . }}/{{ .Values.jarvice.JARVICE_SYSTEM_REPO_BASE }}/jarvice-k8s-nested-scheduler:{{ default .Values.jarvice.JARVICE_IMAGES_TAG .Chart.Annotations.tag }}{{- include "jarvice.version" . }}
```

Comment:
@CJCShadowsan is the `image` override needed long term?

Reply:
It was set to allow us to do overrides for things if we set the JARVICE_KNS_IMAGE_TAG as there were so many changes in there that if we needed an urgent fix we'd not be able to do it easily... That's all it was in there for. 

I think we suggested we could/should do this for all components to allow us to override for situations like hotfixes if I remember correctly...